doc/install.rst

```diff
@@ -266,13 +266,14 @@ Building DLR with TensorFlow C library
 We can use DLR to run Tensorflow 2.x saved models (including `TensorRT converted models <https://blog.tensorflow.org/2021/01/leveraging-tensorflow-tensorrt-integration.html>`_) via TensorFlow C library.
 
 TensorFlow C library can be downloaded from `tensorflow.org <https://www.tensorflow.org/install/lang_c>`_ or built `from source <https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/lib_package/README.md>`_.
+TensorFlow C/C++ headers target can be built from source with `bazel build --config=opt tensorflow:install_headers`.
```

Comment:
We should also inform users that that they can get TensorFlow C/C++ headers folder from Python Tensorflow installation folder - e.g. `/usr/local/lib/python3.7/dist-packages/tensorflow/include`.

Reply:
added.